iset-discouraged

```diff
@@ -444,10 +438,10 @@ New usage of "iseqid3s" is discouraged (3 uses).
 New usage of "iseqm1" is discouraged (3 uses).
 New usage of "iseqp1" is discouraged (10 uses).
 New usage of "iseqp1t" is discouraged (1 uses).
-New usage of "iseqsplit" is discouraged (1 uses).
+New usage of "iseqsplit" is discouraged (0 uses).
```

Comment:
Well the long term goal is to get rid of all the ones which use df-iseq syntax rather than df-seq3 syntax, so it is just a question of how much to do in one pull request.

But sure, looks like I can add removing `iseqsplit` to this pull request without pulling on a thread and having a bunch of other stuff appear. Done.